packages/docs/docs/ssi/did.md

```diff
@@ -4,11 +4,23 @@ sidebar_position: 1
 
 # Decentralized Identifiers (DID)
 
+## Motivation
```

Comment:
Can we move motivation to the Self-Sovereign Identity (SSI) page (parent page of this one)?